spec/controllers/educations_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe EducationsController, type: :controller do
+
+  describe 'GET #index' do
+    # it "populates an array of all educations" do
+    #   monte = create(:education, school: 'Monte')
+    #   ssori = create(:education, school: 'ssori')
+    #   get :index
+    #   expect(assigns(:education)).to match_array([monte, ssori])
+    # end
```

Comment:
I think we should test for this as well! Want to make sure any educations that belong to the user are listed